helper/geojsonify.js

```diff
@@ -67,4 +68,34 @@ function search( docs, params ){
   return geojson;
 }
 
+/**
+ * Copy specified properties from source to dest.
+ * Ignore missing properties.
+ *
+ * @param {object} source
+ * @param {[]} props
+ * @param {object} dest
+ */
+function copyProperties( source, props, dest ) {
+  var count = props.length;
+  for ( var i = 0; i < count; i++ ) {
```

Comment:
@dianashk , not sure if you're soliciting code review, but would a `.forEach()` make sense here? Might make it more immediately clear what this is doing.